### PR TITLE
Improve map.recalculate_bounds! speed

### LINKED ISSUE #2613: Improve map.recalculate_bounds! speed
By @javisantana :

```
# select _postgis_stats('my_table', 'the_geom')::json->'extent';
                      ?column?
-----------------------------------------------------
 {"min":[-5.52764,41.0878],"max":[-3.97387,42.3147]}
```

### PR DESCRIPTION
**Pull request by Kartones**
Closes #2613

### CODE REVIEW

**Comment by Kartones** [comment]
@javisantana CR please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2421/))

File: app/models/map.rb [hunk 1]
```diff
@@ -156,27 +156,50 @@ def update_map_on_associated_entities
...
+    {
+      max: [result[:maxx].to_f, result[:maxy].to_f],
+      min: [result[:minx].to_f, result[:miny].to_f]
+    }
   rescue Sequel::DatabaseError
-    {}
+    default_map_bounds
+  end
+
+  def default_map_bounds
```
Comment (javisantana) [inline app/models/map.rb]: I'd change this to the whole word -90, 90, -180, 180

**Comment by javisantana** [comment]
:es:

File: app/models/map.rb (hunk 1 above)
Comment (Kartones) [inline app/models/map.rb]: I was thinking about it, but left it with the values we're already setting [now](https://github.com/CartoDB/cartodb/blob/master/app/models/map.rb#L39).

**Pull request merged**